Resources/public/templates/aarhus/slides/aarhus-second-sensor-vertical/aarhus-second-sensor-vertical.scss

```diff
@@ -0,0 +1,210 @@
+@import "../../sass/aarhus-second";
+
+.aarhus-second-sensor-vertical {
+  font-size: 16px;
+  @media (orientation: portrait) {
+    font-size: 32px;
+  }
+  position: relative;
+
+  .aarhus-second--container {
+    height: 100vh;
```

Comment:
@yepzdk 
Can we replace vh with something else since it results in scaling issues in the different contexts the slide is in?

Reply:
@tuj Replaced with percentage